traffic_ops/testing/api/v4/statuses_test.go

```diff
@@ -142,13 +145,26 @@ func GetTestStatusesIMS(t *testing.T) {
 }
 
 func CreateTestStatuses(t *testing.T) {
+	response, _, err := TOSession.GetStatuses(client.NewRequestOptions())
+	if err != nil {
+		t.Errorf("could not get statuses: %v", err)
+	}
+	statusNameMap := make(map[string]bool, 0)
+	for _, r := range response.Response {
+		statusNameMap[r.Name] = true
+	}
+
 	for _, status := range testData.Statuses {
-		resp, _, err := TOSession.CreateStatus(status, client.RequestOptions{})
-		if err != nil {
-			t.Errorf("could not create Status: %v - alerts: %+v", err, resp.Alerts)
+		if status.Name != nil {
+			if _, ok := statusNameMap[*status.Name]; !ok {
+				resp, _, err := TOSession.CreateStatus(status, client.NewRequestOptions())
+				t.Log("Response: ", resp)
+				if err != nil {
+					t.Errorf("could not create Status: %v - alerts: %+v", err, resp.Alerts)
+				}
+			}
```

Comment:
You wouldn't need to build this map and do these checks if you just remove standard statuses from the testing fixture data.